Resources/views/CRUD/edit_orm_one_association_script.html.twig

```diff
@@ -47,7 +47,12 @@ This code manage the one-to-many association field popup
                     return;
                 }
 
-                jQuery('#field_container_{{ id }}').replaceWith(html); // replace the html
+                if (jQuery('#field_widget_{{ id }}').children().length) {
+                    var $newRow = jQuery(jQuery.parseHTML(html)).find('tr:last');
```

Comment:
I understand the problem with my solution and a sorted collection. But the problem with recreating the whole form is, that file upload fields can't be filled by javascript or value attribute, so this information will get lost. :(